src/balmy/types.ts

```diff
@@ -15,6 +15,7 @@ export type TokenData = {
   logoURI?: string;
   decimals?: number;
   chainId: ChainId;
+  chainAddresses?: { chainId: ChainId; address: Address }[];
```

Comment:
Just to confirm, this is what you need? This approach would make the payload bigger, since we would have the same array for every instance of the same token. Another approach would be to simply add the coingecko id
But again, what ever best fits your needs

Reply:
We discussed this with @FiboApe and @0xKoaj in the office the other day and concluded that this is the more general solution

Reply:
Yeah we want to keep the tokens as atomic as possible, as in: If I grab a token from the list, I want to have all the information about it available at a first glance, and not have to parse/process other data for it. This will also help us if we later want to propose an EIP for token list schema update were we add this data to the schema